fetcher.go

```diff
@@ -261,7 +262,14 @@ func (p *pool) fetchResource(ctx context.Context, from string, resource string,
 	var resp *http.Response
 	saturnCallsTotalMetric.WithLabelValues(resourceType).Add(1)
 	startReq := time.Now()
-	resp, err = p.config.SaturnClient.Do(req)
+
+	count := 0
+	for ; count < 3; count++ {
```

Comment:
This does help. Proof here:

https://protocollabs.grafana.net/d/6g0_YjBVk/bifrost-caboose-staging?orgId=1&from=now-1h&to=now&editPanel=83

See no impact on latency after deploying this. Let's keep this on.

Reply:
But it's totally unaccountable.
we can't just block on trying 3 times on one node as part of "one attempt"

Reply:
addressed this.